internal/controller/mariadb_controller_replication_test.go

```diff
@@ -155,6 +155,20 @@ var _ = Describe("MariaDB replication", Ordered, func() {
 		By("Expecting to create a secondary Service")
 		Expect(k8sClient.Get(testCtx, mdb.SecondaryServiceKey(), &svc)).To(Succeed())
 
+		By("Expecting role label to be set to primary")
```

Comment:
It will be great to add this checks to the Galera test as well, since this labels are not tied to replication.

Reply:
Added test for galera too.